NEWS.md

```diff
@@ -1,6 +1,8 @@
 # roxygen2 6.1.0.9000
 
 * roxygen2 now specifically imports desc >= 1.2.0 (@crsh, #773, #777, #779)
+* `package_files()` ignores leading whitespace in Collate field (@gustavdelius, 
```

Comment:
Could you please add a newline above this bullet?

Reply:
Done